Assets/Scripts/Minimap.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class Minimap : MonoBehaviour
+    {
+        public Transform player;
+        public GameObject minimap;
+
+        void Update()
+        {
+            if (Input.GetKeyDown(KeyCode.M))
+            {
+                minimap.SetActive(!minimap.activeSelf);
+            }
```

Comment:
Use new InputSystem and actions.

You can add a new action to the already created "PlayerMovement" Action Map. Personally i would use approach with generated C# interfaces, but feel free to choose any other method. The InputSystem is flexible and offers a several different approaches to detect key presses. For instance, you can add a `PlayerInput` component and select method to call on key press

Reply:
Changed to actions. I added new action specifically for minimap